cfme/fixtures/pods_per_ready_status.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from wrapanapi.utils import eval_strings
+
+
+@pytest.fixture(scope='function')
+def pods_per_ready_status(provider):
```

Comment:
OK, so it should be on the Pod object as static method for all pods (since we want just 1 API call)